tests/test_modutils.py

```diff
@@ -296,50 +296,145 @@ class StandardLibModuleTest(resources.SysPathSetup, unittest.TestCase):
     def test_datetime(self) -> None:
         # This is an interesting example, since datetime, on pypy,
         # is under lib_pypy, rather than the usual Lib directory.
-        self.assertTrue(modutils.is_standard_module("datetime"))
+        with pytest.warns(DeprecationWarning):
+            self.assertTrue(modutils.is_standard_module("datetime"))
```

Comment:
While we are changing these lines, would you mind converting them to use `assert`?